bin/update_readme

```diff
@@ -92,6 +93,11 @@ foreach (KnpUOAuth2ClientExtension::getAllSupportedTypes() as $type) {
                 // *should* ? come out looking like valid-ish YAML
                 ? json_encode($child->getDefaultValue())
                 : '{}';
+
+        } elseif ($child instanceof BooleanNode) {
+            $defaultValue = $child->getDefaultValue()
+                ? $child->getDefaultValue()
```

Comment:
Should be `true`

Reply:
do u mean ; `$defaultValue = $child->getDefaultValue() ? 'true'`

Reply:
Not quite, I mean `? 'true'` on this line, i.e.:

```php
 $defaultValue = $child->getDefaultValue()
    ? 'true'
    : 'false';
```

Reply:
that's what I meant ;) sorry just shorted the expression :)
ok I  will do